gcc/rust/resolve/rust-late-name-resolver-2.0.cc

```diff
@@ -130,7 +130,10 @@ Late::visit (AST::IdentifierExpr &expr)
     resolved = label;
   else if (value)
     resolved = value;
-  // TODO: else emit error?
+  else {
+      rust_error_at(expr.get_locus (), "could not resolve identifier expression: %qs", expr.get_ident ().as_string ().c_str ());
+      return;
+  }
```

Comment:
that looks great - for style issues, could you also add braces to the above `if` and `else if` statements? this way the `else` one isn't the only one with braces and it looks more uniform.

Reply:
yes, i'll get on it soon